galactic/distribution.yaml

```diff
@@ -1027,6 +1027,16 @@ repositories:
       url: https://github.com/OUXT-Polaris/embree_vendor.git
       version: master
     status: developed
+  event_array_msgs:
```

Comment:
Please remove all changes to Galactic, since it is end-of-life.

Reply:
done